Source/Core/Core/PowerPC/BreakPoints.cpp

```diff
@@ -78,6 +78,10 @@ void BreakPoints::AddFromStrings(const TBreakPointsStr& bp_strings)
 
     iss >> std::hex >> bp.address;
     iss >> flags;
+    std::getline(iss, bp.message);
```

Comment:
This breaks if you have a breakpoint with no flags, since the `>>` skips an arbitrary amount of whitespace so `flags` gets the first word of the message. Might need to redesign how you store these or rewrite the parsing logic.

Reply:
This shouldn't normally occur unless someone messes up its config file manually because the BP should have either the `break` or `log` flag set. I can add an additional check if you really care. Ditto for MBP which has the same issue.

Reply:
If it's not possible for a breakpoint to have no flags then fair enough. Might still be worth it in case the UI allows this in the future?